Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
> info This package requires node-gyp, which is not currently installed. Yarn will attempt to automatically install it. If this fails, you can run "yarn global add node-gyp" to manually install it.
> error An unexpected error occurred: "/private/tmp/grafana-20170531-29546-atoieg/grafana-4.3.0/src/github.com/grafana/grafana/node_modules/phantomjs-prebuilt: Cannot read property 'config' of undefined".
> info If you think this is a bug, please open a bug report with the information provided in "/private/tmp/grafana-20170531-29546-atoieg/grafana-4.3.0/.brew_home/.config/yarn/global/yarn-error.log".
> info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Maybe `node-gyp` used to be included with npm and no longer is or something like that?